packages/jbrowse-plugin-apollo/src/LinearApolloDisplay/components/ApolloDetails.tsx

```diff
@@ -94,15 +94,16 @@ function AutocompleteInputCell(props: GridRenderEditCellParams) {
 export const ApolloDetails = observer(
   ({ model }: { model: LinearApolloDisplay }) => {
     const session = getSession(model)
+    const { internetAccounts } = getRoot(session) as AppRootModel
+    const apolloInternetAccount = internetAccounts.find(
+      (ia) => ia.type === 'ApolloInternetAccount',
+    ) as ApolloInternetAccountModel | undefined
+    if (!apolloInternetAccount) {
+      throw new Error('No Apollo internet account found')
+    }
     const editable = useMemo(() => {
-      const { internetAccounts } = getRoot(session) as AppRootModel
-      const apolloInternetAccount = internetAccounts.find(
-        (ia) => ia.type === 'ApolloInternetAccount',
-      ) as ApolloInternetAccountModel | undefined
-      if (!apolloInternetAccount) {
-        throw new Error('No Apollo internet account found')
-      }
       return Boolean(apolloInternetAccount.getRole()?.includes('user'))
+      // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
What was the reason for this change? I'd like to not change it if possible, since I don't want to run `find` on all accounts every render.

Reply:
I simply forgot to restore it yesterday after I did some tests. Sorry for that. There is no need to change anything in ApolloDetails.tsx